src/shared/components/StyledSlider/index.jsx

```diff
@@ -1,14 +1,17 @@
 import { styled } from '@mui/material/styles'
 import Slider from '@mui/material/Slider'
 
+const celadonColor = getComputedStyle(document.documentElement).getPropertyValue('--celadon')
```

Comment:
Wow, this is not the intended way of doing it. I guess you had troubles with using css variable in the js code. You can do it either way:
- create js variable with this colour and use it
- define new color in theme and use it through props, like this
```
export const StyledSlider = styled(Slider)((theme) => ({
  color: theme.celadonColor,
  ```
  And naming colors with color names is not good option. Imagine that your designer wants to change color from celadon to purple? What do you do? You change the value of the celedon variable, and the name is not valid rn. You should use names like `primaryColor` `secondaryColor`, `mainColor`, `backgroundBaseColor`, `textContrastPrimary` etc. You can read more about theming in the material ui docs